unmet dependency angular/cdk

#### Bug, feature request, or proposal:
BUG

#### What is the expected behavior?
ng serve should work

#### What is the current behavior?
compilation breaks:

```
ERROR in /Applications/MAMP/htdocs/memberhive2/web/node_modules/@angular/material/typings/core/core.d.ts (1,61): Cannot find module '@angular/cdk'.

ERROR in ./~/@angular/material/@angular/material.es5.js
Module not found: Error: Can't resolve '@angular/cdk' in '/Applications/MAMP/htdocs/memberhive2/web/node_modules/@angular/material/@angular'
 @ ./~/@angular/material/@angular/material.es5.js 19:0-75
 @ ./src/app/app-material.module.ts
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi webpack-dev-server/client?http://localhost:4200 ./src/main.ts
webpack: Failed to compile.
```
During `yarn upgrade` you get the following warning:
`warning "@angular/material@2.0.0-beta.7-51bf26e" has unmet peer dependency "@angular/cdk@0.0.0-PLACEHOLDER".`

@DevVersion it may have something to do with 17cc4a198f336e8ebefbb2710d8e80df2595db7e?

#### What are the steps to reproduce?

Just open the Plunker example ... the console tells the story: https://goo.gl/DlHd6U

#### What is the use-case or motivation for changing an existing behavior?
continue with upgrades

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
TS 2.4.0
@angular/cli: 1.1.3
node: 8.0.0
os: darwin x64
@angular/animations: 4.2.4
@angular/common: 4.2.4
@angular/compiler: 4.2.4
@angular/core: 4.2.4
@angular/flex-layout: 2.0.0-rc.1
@angular/forms: 4.2.4
@angular/http: 4.2.4
@angular/material: 2.0.0-beta.7-51bf26e
@angular/platform-browser: 4.2.4
@angular/platform-browser-dynamic: 4.2.4
@angular/router: 4.2.4
@angular/cli: 1.1.3
@angular/compiler-cli: 4.2.4
@angular/platform-server: 4.2.4

#### Is there anything else we should know?
I am pulling from the master branch, updating almost daily. Yesterday morning the update did not break the build.

I get the same error.
for what its worth: i get the following errors after an npm install which includes @angular/material@2.0.0-beta.7-51bf26e
`npm WARN @angular/material@2.0.0-beta.7-51bf26e requires a peer of @angular/cdk@0.0.0-PLACEHOLDER but none was installed.

up to date in 3.194s
PS C:\dev\material2-start> npm install @angular/cdk@0.0.0-PLACEHOLDER
npm ERR! code E404
npm ERR! 404 Not Found: @angular/cdk@0.0.0-PLACEHOLDER

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Tom Devolder\AppData\Roaming\npm-cache\_logs\2017-06-23T07_26_44_976Z-debug.log`

--- a/scripts/deploy/publish-build-artifacts.sh
+++ b/scripts/deploy/publish-build-artifacts.sh
@@ -55,6 +55,10 @@ publishPackage() {
   # Update the package.json version to include the current commit SHA.
   sed -i "s/${buildVersion}/${buildVersion}-${commitSha}/g" package.json
 
+  # For build artifacts the different Angular packages that refer to the 0.0.0-PLACEHOLDER should
+  # be replaced with the Github builds that are published at the same time.
+  sed -i "s/0.0.0-PLACEHOLDER/${buildVersion}-${commitSha}/g" package.json
+
   # Prepare Git for pushing the artifacts to the repository.
   git config user.name "${commitAuthorName}"
   git config user.email "${commitAuthorEmail}"

